### ISSUE TITLE
Shitcoins that don't work

### ISSUE THREAD

**Issue by DLDLightwave** [bonesoul/CoiniumServ#877]
11/26/2017 20:25:17 +00:00 [Debug] [ShareManager] [Bitcoin-Scrypt] We thought a block was found but it was rejected by the coin daemon; [639ca152d2b6c18cb01950b276959e8c4b847acc1ab564d98ecb6bfac4aed126] - reason; Method not found

11/26/2017 18:05:39 +00:00 [Debug] [ShareManager] [MurrayCoin] We thought a block was found but it was rejected by the coin daemon; [f3fabe2b478ecf58df5382fc51ced87f16d36270c41c7d8e4aadfc06842a1441] - reason; Block not found

11/25/2017 04:54:29 +00:00 [Debug] [ShareManager] [FujiCoin] We thought a block was found but it was rejected by the coin daemon; [b9c10372dbf5a383fac4bce77cc2a69dedf39bb08614f695358e84f2e5b0e045] - reason; Block not found

**Comment by DLDLightwave** [comment]
11/26/2017 21:01:44 +00:00 [Debug] [ShareManager] [42] We thought a block was found but it was rejected by the coin daemon; [83eb7f541dcd24ba9c42fac55300a036f68f8cc385904bb5dadc45f93ecbd1e1] - reason; Block not found

**Comment by bonesoul** [comment]
Some shitcoins may need special changes to codebase, that could be the case.

**Comment by DLDLightwave** [comment]
11/30/2017 19:54:50 +00:00 [Debug] [ShareManager] [BeezerCoin (BEEZ)] We thought a block was found but it was rejected by the coin daemon; [18bf8b9688ee0f1502cbaede6979154df520be6dfb2775a4d408053eef7e7d0f] - reason; Block not found


what can be done to help figure out why these dont work as, asbest I can tell there is nothing special about them.

**Comment by bonesoul** [comment]
It's too hard to debug here if we miscalculated a coinbase tx or the the block itself was a orphan one.

**Comment by DLDLightwave** [comment]
12/02/2017 15:53:50 +00:00 [Debug] [ShareManager] [TopCoin (TOP)] We thought a block was found but it was rejected by the coin daemon; [fee7027b5ee9e702e6b6181bee6fd26d9c85518e6082d7970fc467613f93e039] - reason; Block not found

I'm fairly sure these aren't orphans btw... I've tested long enough to discount that, and am ona fast network.

**Comment by bonesoul** [comment]
Being on a fast network doesn't guarantee you'll not ever hit orphans.

**Comment by DLDLightwave** [comment]
Yeah... but I  block notify from the coin daemon and have clearly seen: someone else find a block, CoiniumServ issue a new job, and then get the errors displayed without another block being found for a while.  I'm also just posting the error once, but have seen them dozens/hundreds of times for each coin tested.
Or simply starting CoiniumServ after a block is found, then the errors are logged before another block is found.

This post is  more for someone wondering why a coin they are trying to mine might not work.

**Comment by regtable** [comment]
add txmessages = true  or what ever it says in the config as optional parameters. systematically add these including the coinbase for the "template" this helped me with some coins others not at all. mainly though i see this error when a block is not found or conflicted with the network. its coiniumserv's way of telling you it was an orphan block. 
i see it maybe 3x for ever 1 found block on some coins :-)

**Comment by regtable** [comment]
also some pos coins require setting up to know it is pos. this is because the block templates are different. so maybe the wallet is rejecting due to this. check debug.log for the daemon oi bet it says somethign along the line of coinbase too low. i.e  output is smaller than expected block size something else i hit on meaning every block i mine with incakoin is missing 0.1 off the top never understood why

**Comment by DLDLightwave** [comment]
Thanks. Coinmarketcap shows these some of these are mineable... so I don;t think they are PoS.
fwiw / I have never seen a coinbase too low or warning about the output size for these coins.

I will definitely try the config tweaks tho...

**Comment by DLDLightwave** [comment]
SetBestChain: new best=00000000fec0600aeb0e8ecfc37b22110f242d2c15d770dc7f004ae4beac0f02  height=1775864  trust=253531475938210  date=12/10/17 07:12:15
ThreadRPCServer method=getblock
received getdata for: block 00000000fec0600aeb0e

It seems like it should be doing:
topcoin3d getblock 00000000fec0600aeb0e8ecfc37b22110f242d2c15d770dc7f004ae4beac0f02
{
    "hash" : "00000000fec0600aeb0e8ecfc37b22110f242d2c15d770dc7f004ae4beac0f02",
    "confirmations" : 0,
    "size" : 324,
    "height" : 1775864,
    "version" : 4,
    "merkleroot" : "82944a51dc99de17bfbc6d0059ec0d30f6bba1df6e2139d1e53a8705b8c64b83",
    "mint" : 2.50000000,
    "time" : 1512889935,
    "nonce" : 3909391143,
    "bits" : "1d02197a",
    "difficulty" : 0.47629257,
    "previousblockhash" : "0000000081ad8a196ff95f09bdb9d95d9eba9ba1623c3264a77e85b218ca0a3a",
    "flags" : "proof-of-work",
    "proofhash" : "00000000fec0600aeb0e8ecfc37b22110f242d2c15d770dc7f004ae4beac0f02",
    "entropybit" : 0,
    "modifier" : "ea0e47e5e0b1d009",
    "modifierchecksum" : "20a3ebdc",
    "tx" : [
        "82944a51dc99de17bfbc6d0059ec0d30f6bba1df6e2139d1e53a8705b8c64b83"
    ],
    "signature" : "3045022100c71850ae51e0d9b2a5419083fa91de70f884732455e5429bf49e95cf79da5d250220731e479c9fd1b500b7ec91235882f05e79c496518d3c2c0049751aaad60a4799"
}


CoiniumServ doesnt think I won the block, though I did.
As it cant find the block... it seems to be asking for the wrong value though...
Is this one easier to fix?

**Comment by Maleksh** [comment]
How can we enable this Block notify in the CoiniumServ Pool.
it is very hard to find a block without this feature enabled
